binaries/mbc.cpp

```diff
@@ -622,8 +625,8 @@ int main(int argc, char *argv[])
         exit(1);
     }
 
-    DB::SetLogFile(std::string(db_dir) + "/mabain.log");
-    DB::LogDebug();
+    // DB::SetLogFile(std::string(db_dir) + "/mabain.log");
```

Comment:
Need to comment the CloseLogFile call at the end.

Reply:
done